src/embedResources.ts

```diff
@@ -36,6 +36,10 @@ export function embed(
     ? resolveUrl(resourceURL, baseURL, document, window)
     : resourceURL
 
+  if (options.skipURLs && options.skipURLs.includes(resourceURL)) {
```

Comment:
You can use the optional chaining feature.
`if(options.skipURLs?.includes(resourceURL))`

Reply:
done